db/migrate/20251112121438_remove_place_of_worship_start_date_to_job_applications.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePlaceOfWorshipStartDateToJobApplications < ActiveRecord::Migration[8.0]
+  def change
```

Comment:
Think you need to add the old migration as well, so the column is there to remove